fragment.fs

```diff
@@ -0,0 +1,16 @@
+uniform sampler2D u_Texture;
+uniform float u_Threshold;
+
+varying vec2 v_UV;
+
+void main()
+{
+    vec4 sample = texture2D(u_Texture, v_UV);
```

Comment:
I don't understand this. It looks like a bunch of uninitialized variables being used.